docs/contributing/the_basics.md

```diff
@@ -10,37 +10,27 @@ You can use any operating system.
 Install all development dependencies using:
 
 ```console
-$ pipenv install --dev
-$ pipenv shell
-$ pre-commit install
-```
-
-If you haven't used `pipenv` before but are comfortable with virtualenvs, just run
-`pip install pipenv` in the virtualenv you're already using and invoke the command above
-from the cloned _Black_ repo. It will do the correct thing.
-
-Non pipenv install works too:
-
-```console
-$ pip install -r test_requirements.txt
-$ pip install -e .[d]
+$ python3 -m venv .venv
```

Comment:
```suggestion
$ python3 -m venv .venv
$ source .venv/bin/activate
```
(Or something else on Windows?)

Reply:
Ah, having to activate manually... I use [`vf`](https://virtualfish.readthedocs.io/en/latest/) these days so forgot about it. Windows users probably know how to create and activate their venvs. And those that don't we probably don't want to encourage to use Black from `main` (at least not **yet**!).

Reply:
I think it would suffice to mention just to use any virtual environment. Any user can then see the commands for their specific stuff. The deps are anyways installed from requirements. Or is there a difference I'm not aware of?